mesmerize_napari/main_offline_gui.py

```diff
@@ -411,20 +411,35 @@ def view_downsample_mcorr(self):
     def view_shifts(self):
         s = self.selected_series()
         if s["params"]["mcorr_kwargs"]["pw_rigid"]:
-            x_shifts, y_shifts = s.mcorr.get_shifts(output_type="napari-1d")
-            self.viewer.add_image(x_shifts, name=f'{s["name"]}: X shifts')
-            self.viewer.add_image(y_shifts, name=f'{s["name"]}: Y shifts')
+            shifts = s.mcorr.get_shifts()
```

Comment:
you should've checked out a new branch from master for this :P

Reply:
Ah shit I thought I did, my bad!